ScreenManager.lua

```diff
@@ -47,11 +52,65 @@ local function clear()
     end
 end
 
+---
+-- Check if the screen is valid or error if not
+--
+local function validScreen( screen )
```

Comment:
I didn't use `is` because it doesn't return a boolean telling you if it is valid or not but actually errors if invalid.